lib/trip_updates/trip_updates_pub_sub.ex

```diff
@@ -0,0 +1,102 @@
+defmodule TripUpdates.TripUpdatesPubSub do
+  @moduledoc """
+  Publish a list of blocked routes to subscribers.
+  """
+  use GenServer
+  alias TripUpdates.{StopTimeUpdate, TripUpdates}
+
+  defstruct block_waivered_routes: []
```

Comment:
suggestion: I think it could be useful to define a `t` typespec for this state struct. That would help a reader quickly understand what kind of data to expect. There are also a couple of places throughout this file where this data is defined as `any`, but with such a definition, they could be changed to `t()`.